File: app/src/main/java/ch/abler/aline/alarmdroid/MainActivity.java
package ch.abler.aline.alarmdroid;

import android.accounts.Account;
import android.accounts.AccountManager;
import android.content.Context;
import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;
import android.util.Log;

public class MainActivity extends AppCompatActivity {

    public static final String AUTHORITY = "ch.abler.aline.alarmdroid.provider";
    public static final String ACCOUNT_TYPE = "aline.abler.ch";
    public static final String ACCOUNT = "dummyaccount";
    Account account;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
    }

    private static void debug(String msg) {
        Log.d("MainActivity", msg);
    }
}

File: app/src/main/java/ch/abler/aline/alarmdroid/MainActivity.kt
package ch.abler.aline.alarmdroid

import android.app.AlarmManager
import android.app.Service
import android.support.v7.app.AppCompatActivity
import android.os.Bundle
import android.widget.TextView

class MainActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
        text = findViewById(R.id.main_text)
        val alarmManager = getSystemService(Service.ALARM_SERVICE) as AlarmManager
        val alarmClockInfo = alarmManager.nextAlarmClock
        val data = alarmClockInfo?.triggerTime.toString()

        text.text = data

    }

    companion object {
        lateinit var text: TextView
    }
}
